Makefile

```diff
@@ -1,6 +1,6 @@
 GOBIN ?= ${GOPATH}/bin
 IMG   ?= tackle2-addon-windup:latest
-CONTAINER_RUNTIME := $(shell command -v podman 2> /dev/null || echo docker)
+CONTAINER_RUNTIME := docker
```

Comment:
Same as https://github.com/konveyor/tackle2-hub/pull/177#discussion_r1000898275 either we "discover" it or drop it entirely.

Reply:
Added separate target for docker and podman